internal/service/ipam/api_dhcp_host_data_source_test.go

```diff
@@ -0,0 +1,108 @@
+package ipam_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/infobloxopen/bloxone-go-client/ipam"
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/acctest"
+)
+
+/*
+TODO:
+// Add unit tests for DHCP config profiles
+*/
+
+func TestAccDhcpHostDataSource_Filters(t *testing.T) {
+	dataSourceName := "data.bloxone_dhcp_hosts.test"
+	var v ipam.IpamsvcHost
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDhcpHostDataSourceConfigFilters("TF_TEST_HOST_01"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDhcpHostExists(context.Background(), dataSourceName, &v),
+					resource.TestCheckResourceAttr(dataSourceName, "results.#", "1"),
+					resource.TestCheckResourceAttr(dataSourceName, "results.0.name", "TF_TEST_HOST_01"),
+					resource.TestCheckResourceAttr(dataSourceName, "results.0.id", "dhcp/host/465008"),
```

Comment:
ID shouldn't be hard coded. You can simply check if ID is set or not. Or you can compare with `v.Id`